code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -1215,6 +1215,7 @@
 			/obj/item/storage/holster/t19 = -1,
 			/obj/item/storage/holster/blade/machete/full = -1,
 			/obj/item/storage/holster/blade/machete/full_harvester = -1,
+			/obj/item/weapon/twohanded/glaive/harvester = -1,
```

Comment:
?

Reply:
Something something I didn't pull master I think.
I'll fix this now.